CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
-# Changelog
+Changelog
```

Comment:
This is a markdown file - was the change to the header intentional?

Reply:
Good question, I don't think it was intentional? I removed it and it works fine